src/main/java/org/apache/commons/compress/archivers/zip/ZipUtil.java

```diff
@@ -30,11 +33,32 @@
  * @Immutable
  */
 public abstract class ZipUtil {
+
+    /**
+     * <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms724290%28v=vs.85%29.aspx">Windows File Times</a>
+     * <br>
```

Comment:
No br tag needed, open and close a p tag.

Reply:
Fixed and moved this to TimeUtils